src/Files.Sdk.Storage/Enums/CreationCollisionOption.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Files.Sdk.Storage.Enums
+{
+    public enum CreationCollisionOption : byte
```

Comment:
```suggestion
    public enum CollisionOption : byte
```
Can this be simplified?

Reply:
There's also NameCollisionOption which has some key differences.